tests/e2e/cjs/index.test.js

```diff
@@ -266,3 +280,29 @@ describe('IO', async () => {
     assert.ok(tokenCost);
   });
 });
+
+describe('ANTRegistry', async () => {
+  let registry;
+  let wallet;
+  let address;
+  let signer;
+
+  before(async () => {
+    wallet = await arweave.wallets.generate();
+    address = await arweave.wallets.jwkToAddress(wallet);
+    const arbundlesSigner = new ArweaveSigner(wallet);
+    signer = createAoSigner(arbundlesSigner);
+    registry = ANTRegistry.init({ signer: arbundlesSigner });
+  });
+
+  it('should deploy and register a new ANT', async () => {
+    const antId = await spawnANT({
+      signer,
+    });
+    await registry.register({ processId: antId });
+    await new Promise((resolve) => setTimeout(resolve, 5000));
+    const antIdsRes = await registry.accessControlList({ address });
+    const antIds = [...antIdsRes.Owned, ...antIdsRes.Controlled];
+    assert(antIds[0] == antId);
+  });
```

Comment:
just do a read test here - these tests are to ensure the apis work in the respective environment, we should not actually write any messages to the process from these